engine/src/main/scala/cromwell/webservice/routes/wes/WesRunRoutes.scala

```diff
@@ -0,0 +1,79 @@
+package cromwell.webservice.routes.wes
+
+import akka.actor.ActorRef
+import akka.http.scaladsl.model.{HttpHeader, StatusCodes}
+import akka.http.scaladsl.server.Directives._
+import akka.http.scaladsl.server.Route
+import akka.http.scaladsl.server.directives.RouteDirectives.complete
+import akka.util.Timeout
+import WesRunRoutes._
+import akka.stream.ActorMaterializer
+import com.typesafe.config.ConfigFactory
+import cromwell.webservice.routes.MetadataRouteSupport.metadataQueryRequest
+
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.concurrent.Future
+import scala.util.{Failure, Success}
+
+trait WesRunRoutes {
+  implicit def materializer: ActorMaterializer
+
+  implicit val timeout: Timeout
+
+  val serviceRegistryActor: ActorRef
+
+  lazy val runRoutes: Route =
+      pathPrefix("ga4gh" / "wes" / "v1") {
+        concat(
+          pathPrefix("runs") {
+            concat(
+              pathEnd {
+                concat(
+                  get {
+                    parameters(("page_size".as[Int].?, "page_token".?)) { (pageSize, pageToken) =>
+                      completeCromwellResponse(listRuns(pageSize, pageToken, serviceRegistryActor))
+                    }
+                  }
+
+                )
+              }
+            )
+          }
+        )
+      }
```

Comment:
The three `concat()` invocations here do nothing. Their purpose is to concatenate two or more routes together, but they each have exactly one route in them.

```suggestion
  lazy val runRoutes: Route =
    pathPrefix("ga4gh" / "wes" / "v1") {
      pathPrefix("runs") {
        pathEnd {
          get {
            parameters(("page_size".as[Int].?, "page_token".?)) { (pageSize, pageToken) =>
              completeCromwellResponse(wes2CromwellInterface.listRuns(pageSize, pageToken, cromwellRequestHeaders, serviceRegistryActor))
            }
          }
        }
      }
    }
```

Reply:
Looks like this was fixed in one recent commit and the fix undone in another

Reply:
Looks like this was fixed in one recent commit and the fix undone in another